lib/Horde/Argv/Values.php

```diff
@@ -24,60 +24,152 @@
  * @copyright 2010-2017 Horde LLC
  * @license   http://www.horde.org/licenses/bsd BSD
  */
+
```

Comment:
TBH I accepted this because it works but for a value object we're probably better off making it a descendend of stdClass. Even worse, this has barely any functionality not provided by ArrayObject.